test/types/partial/partial-no-q-warning.chpl

```diff
@@ -20,7 +20,7 @@ record r3 {
 }
 
 type t3 = r3(real);
-var z: t3(int);
+//var z: t3(int); // currently an error
```

Comment:
Is this test still interesting/valuable with this commented out?  (it doesn't look like it does much).  Should it be converted into a future (of some sort) instead?

Reply:
The .good file shows lots of warnings for partial instantiations missing trailing `?`, on lines 6, 14, 22, 37, 45. Note that the test is testing this warning that still appears on line 22 even though we never default initialized `z`.